server/core/admin.cc

```diff
@@ -279,6 +280,38 @@ static bool load_ssl_certificates()
     const auto& cert = config.admin_ssl_cert;
     const auto& ca = config.admin_ssl_ca_cert;
 
+    const auto& cipher = config.admin_ssl_cipher;
+
+    // Set cipher appropriately, including some shorcuts.
+    if (cipher.empty())
```

Comment:
A separate function that does the translation into GnuTLS values would make this cleaner. When the value of `config.admin_ssl_cipher` is changed into an enumeration, this becomes quite simpler to do.

In addition, SSLV3 is not supported by MaxScale and should not be supported by the REST API.